pkg/cli/loop_test.go

```diff
@@ -126,42 +126,3 @@ func quitOn(lp *loop, retTrigger event, ret string, err error) handleCb {
 		}
 	}
 }
-
```

Comment:
Don't remove this; example functions are run as test cases by `go test`.

Reply:
Ah! Another day where I learned something new. Reinstated but with the function name mangled so `go vet` doesn't complain that it can't find a `Loop` symbol and a comment to aid future readers who might be unaware of this `go test` feature.